invokeai/app/services/session_queue/session_queue_sqlite.py

```diff
@@ -30,20 +31,25 @@
 
 class SqliteSessionQueue(SessionQueueBase):
     __invoker: Invoker
+    __app_config: InvokeAIAppConfig
     __conn: sqlite3.Connection
     __cursor: sqlite3.Cursor
     __lock: threading.RLock
 
     def start(self, invoker: Invoker) -> None:
         self.__invoker = invoker
         self._set_in_progress_to_canceled()
-        prune_result = self.prune(DEFAULT_QUEUE_ID)
+        if self.__app_config.empty_queue_on_startup:
```

Comment:
You can access this using `self.__invoker.services.configuration.empty_queue_on_startup`, no need to change the constructor